src/drive/web/modules/public/Actions.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
```

Comment:
Most of this file is lifted from https://github.com/cozy/cozy-drive/blob/master/src/drive/web/modules/drive/Container.jsx — I didn't spend too much time on it because it's been rewritten in #1979